DEVELOPMENT.md

```diff
@@ -65,3 +65,22 @@ This is especially important so that papermill can run smoothly.
  table in the README, ideally grouping the newly added example with any other related examples.
 
 - After a new notebook has been added and pushed to `master` branch, AVOID changing the notebook and folder names, as this may break existing links referencing the example notebook throughout cleanlab documentation, blog posts, and more.
+
+
+### Running all examples
+
```

Comment:
was there comment to add that this only works in certain python version?

Reply:
Was adding it the same time you made the comment!